src/map/script.c

```diff
@@ -9133,6 +9133,92 @@ static BUILDIN(grouprandomitem)
 	return true;
 }
 
+/*==========================================
+ * utils for item group
+ *------------------------------------------*/
+static bool utils_itemgroup_remove_duplicate(struct item_group *group)
```

Comment:
This should probably be added to interface and change the name to reflect the interface `script`